packages/modules/auth/src/services/auth-module.ts

```diff
@@ -121,6 +121,36 @@ export default class AuthModuleService
     return Array.isArray(data) ? serializedUsers : serializedUsers[0]
   }
 
+  // @ts-expect-error
+  createProviderIdentities(
+    data: AuthTypes.CreateProviderIdentityDTO[],
+    sharedContext?: Context
+  ): Promise<AuthTypes.ProviderIdentityDTO[]>
+
+  createProviderIdentities(
+    data: AuthTypes.CreateProviderIdentityDTO,
+    sharedContext?: Context
+  ): Promise<AuthTypes.ProviderIdentityDTO>
+
+  @InjectManager("baseRepository_")
+  async createProviderIdentities(
+    data:
+      | AuthTypes.CreateProviderIdentityDTO[]
+      | AuthTypes.CreateProviderIdentityDTO,
+    @MedusaContext() sharedContext: Context = {}
+  ): Promise<AuthTypes.ProviderIdentityDTO | AuthTypes.ProviderIdentityDTO[]> {
+    const providerIdentities = await this.providerIdentityService_.create(
+      data,
+      sharedContext
+    )
+
+    return await this.baseRepository_.serialize<
+      AuthTypes.ProviderIdentityDTO[]
+    >(providerIdentities, {
+      populate: true,
```

Comment:
IIRC you don't need populate, as it will do it by default.

Reply:
Yes you are right